### ISSUE TITLE
Better Logging and Errors

### ISSUE THREAD

**Issue by DanielWright** [DynamoMTL/ship_fosdick#12]
TBD

**Comment by peterberkenbosch** [comment]
Adding standard Rails.logger? 
Errors will bubble up when processing though.

**Comment by DanielWright** [comment]
At a base level, I was thinking of at least a counter of shipments pushed, much as the Wombat client used to do:

```
Pushed Shipments: 453
Errors: 2
```

Beyond that, I think it's important to integrate a backoffice dashboard which surfaces errors a bit to admins. It'll be useful for us to look at backtraces on Sentry, but something a little more user-friendly will need to be made available for customer-service purposes.

**Comment by peterberkenbosch** [comment]
In the current implementation we push per shipment, there is no concept of batches. A shipment is either successfully pushed or not. We could wrap some counters in the rake task where this is implemented maybe.

Personally I would not add any UI here so we can keep this gem light and focussed on the core functionality, communication with fosdick.
when using sidekiq for example there will be an dashboard that will show the jobs that did not ran successfully.

**Comment by DanielWright** [comment]
Yep, you're right, delegating that responsibility to the host app makes sense.
